infrastructure/resource_group_level/application-gateway-portal.bicep

```diff
@@ -0,0 +1,287 @@
+param location string
+param environment string
+param gatewaySubnetId string
+param tfsIpId string
+param portalApiHostname string
+param portalUiHostname string
+param gefUiHostname string
+param apiPortalAccessPort int = 0
+
+param applicationGatewaySku object = {
+  name: 'WAF_v2'
+  tier: 'WAF_v2'
+}
+
+param autoscaleConfiguration object = {
+  minCapacity: 1
+  maxCapacity: 5
+}
```

Comment:
I've made some of the values parametrisable but defaulted. There are some others in theory, but I think they are ok inlined and could easily be updated later.